src/simtools/schemas/application_workflow.metaschema.yml

```diff
@@ -0,0 +1,244 @@
+                model versions."
+            CONFIGURATION:
+              "$ref": "#/definitions/Configuration"
+            INTEGRATION_TESTS:
+              type: array
+              items:
+                "$ref": "#/definitions/IntegrationTest"
+          required:
+            - APPLICATION
+            - CONFIGURATION
+    title: CTASimpipe
+  Configuration:
+    description: "Command line configuration of simtools application."
+    type: object
+    additionalProperties: true
+    title: Configuration
+  IntegrationTest:
+    description: "List of integration tests."
+    type: object
+    additionalProperties: false
+    properties:
+      OUTPUT_FILE:
+        description: |
+          "Path of output file tested to be generated by integration test."
+        type: string
+      TEST_OUTPUT_FILES:
+        type: array
+        items:
+          type: object
+          properties:
+            PATH_DESCRIPTOR:
+              type: string
+            FILE:
+              type: string
+            EXPECTED_OUTPUT:
+              description: |
+                "Expected output of integration test."
+              type: object
+          required: ["PATH_DESCRIPTOR", "FILE"]
+      FILE_TYPE:
+        description: |
+          "Expected file type of output file generated by integration test."
+        type: string
+      REFERENCE_OUTPUT_FILE:
+        description: |
+          "Reference file used for comparison."
+        type: string
+      TEST_SIMTEL_CFG_FILES:
+        description: |
+          "Reference file used for comparison of sim_telarray configuration files."
+        type: object
+        additionalProperties: false
+        patternProperties:
+          "^[0-9]+\\.[0-9]+\\.[0-9]+$":  # Semantic versioning pattern (e.g., "5.0.0", "6.0.0")
+            type: string
+            description: Path to the configuration file for the given version.
+        minProperties: 1
+      TOLERANCE:
+        description: "Allowed tolerance for floating point comparison."
+        type: number
+      TEST_COLUMNS:
+        description: "List of columns to be compared, each with optional cut conditions."
+        type: array
+        items:
+          type: object
+          properties:
+            TEST_COLUMN_NAME:
+              description: "Name of the column to compare."
+              type: string
+            CUT_COLUMN_NAME:
+              description: "Name of the column for applying the cut."
+              type: string
+            CUT_CONDITION:
+              description: "Condition to apply to the cut column."
+              type: string
+    title: ApplicationWorkflow
+...
+---
+$schema: http://json-schema.org/draft-06/schema#
+$ref: '#/definitions/SimtoolsApplicationWorkflowConfiguration'
+title: SimPipe application workflow configuration metaschema
+description: YAML representation of application workflow configuration metaschema
+version: 0.2.0
```

Comment:
Where do we use 0.2.0, I can't see it?

Reply:
You are right, we don't use it anywhere anymore. So I probably should remove the 0.2.0 definition?

Reply:
Yes, I think it can be removed.